data/mods/Xedra_Evolved/items/tools.json

```diff
@@ -63,5 +63,23 @@
       }
     ],
     "flags": [ "ALLOWS_REMOTE_USE" ]
+  },
+  {
+    "id": "sacrifice_table",
+    "type": "TOOL",
+    "category": "tools",
+    "name": { "str": "sacrifice table" },
+    "description": "A magickal space where offerings are sacrificed and transformed.",
```

Comment:
*Not sure if using magick or magic as basic word*

Reply:
Ever since I read the sentence, "You should never trust anyone who spells magick with a k." in the early nineties, I've leaned towards spelling it with a k whenever it's my choice.